.gitignore

```diff
@@ -74,3 +74,5 @@ yarn-error.log
 .phpunit.result.cache
 /phpunit.xml
 ###< symfony/phpunit-bridge ###
+
+local-php-security-checker
```

Comment:
Good DID measure to ensure we never commit this binary! 

What we also do, and will implement in the Travis -> GHA migration of the QA tools, is to remove the binary after running the test.